### DISCUSSION TITLE
请问一下如何给编译目录所在的空间瘦身

### DISCUSSION THREAD

**Discussion by GreatMichaelLee** [coolsnowwolf/lede#12532] [Bugs / 问题反馈]
如题，之前拉完代码配置好，新编译的时候才不超过50个G的空间，现在一下涨到快100个G了，也不知道哪里占空间，谁能指导一下吗
我尝试过如下的方式（前提是不要让我清掉配置重配）但清理不多。
1. rm -rf ./tmp && make clean (能好一些，但清理的不多）
2. 删除/dl目录重新make download (也不多）

**Comment by realabertian** [comment]
遇到一样的问题, 我的 wsl2 的空间满了 编译都报错了....
